cmake/platform/arm-linux-gnueabihf.cmake

```diff
@@ -5,5 +5,17 @@
 # toolchain file found at [../toolchain/raspberrypi.cmake](../toolchain/raspberrypi.cmake).
 ##
 
+# Set platform default for stubbed drivers
+if (NOT DEFINED FPRIME_USE_STUBBED_DRIVERS)
+   set(FPRIME_USE_STUBBED_DRIVERS OFF)
+endif()
+
+# Set platform default for baremetal scheduler drivers
+if (NOT DEFINED FPRIME_USE_BAREMETAL_SCHEDULER)
+   set(FPRIME_USE_BAREMETAL_SCHEDULER OFF)
```

Comment:
Is the `FPRIME_USE_BAREMETAL_SCHEDULER=OFF` correct here?

Reply:
yes.  In all in-repo cases, we do not run the scheduler.  We'd use the scheduler on the Arduino/teensyduino packages of fprime-community.